### ISSUE TITLE
Top level array element

### ISSUE THREAD

**Issue by thefatoneinthecorner** [tulios/json-viewer#119]
Hi,

It appears that JSON viewer ignores JSON where the top level is an array. Is this correct?

-- Alastair

**Comment by tulios** [comment]
Hi @thefatoneinthecorner, sorry for the delay. It should work, I have a test file for it and it is working:

![screen shot 2016-11-29 at 09 35 48](https://cloud.githubusercontent.com/assets/33231/20702240/4b2c6bee-b617-11e6-9a48-41766f449ef9.png)

Can you check this URL:
https://raw.githubusercontent.com/tulios/json-viewer/master/tests/test.array.json

It might be that the API you are using are returning `text/html` as a content-type, json-viewer is not based on content-type but rather if chrome knows how to render the content or not and when it is `text/html` the browser tries to render it.

**Comment by thefatoneinthecorner** [comment]
Hi Túlio,

You are quite correct! And I am delighted - json-viewer is making a big difference for me.

It’s strange because I thought I did everything right regarding restarting Chrome, setting the content-type, but it didn’t work. I then changed the top level element from an array to an object and it started working. I did some research and apparently, strictly speaking, the json top level element should be an object. I assumed that was the problem ...

Anyway, it all works now and I am very happy. Thanks for your help and a great piece of code!


— Alastair
